packages/web-forms/e2e/README.md

```diff
@@ -0,0 +1,58 @@
+# E2E Tests for `@getodk/web-forms`
+
+This directory contains end-to-end (E2E) tests for the `@getodk/web-forms` package, which powers form filling and submission editing of ODK forms in a web browser. These tests use [Playwright](https://playwright.dev/) to simulate user interactions and ensure the package works as expected in real-world scenarios.
+
+## Folder Structure
+
+The E2E tests are located in `packages/web-forms/e2e/`. Here's the structure and purpose of each directory:
+
+```
+e2e/
+├── fixtures/         # Sample forms and data for testing.
```

Comment:
At the moment, this folder is empty, but next time, it'd be good to take the XML forms from here. I added a ToDo comment for this improvement in the PreviewPage.ts

Reply:
Thanks for the context!

I'm still unsure if the fixtures in `@getodk/common` will fully meet the E2E testing needs, but if a special case arises, e.g., verify UX based on special appearance that modifies the display, I don’t think there will be many, and we can add it to`@getodk/common`. 

This was intended to keep it contained from other types of testing, but I don't have a strong preference. This approach has existed for a while without producing any issues with other test types. I'm open to keeping it that way. 

Do you agree to remove the fixture from these e2e tests and keep using the one in `@getodk/common`?

Reply:
Makes sense to me to remove the `fixture` directory from here and document that form fixtures should be put in `@getodk/common` so they can be used in different contexts and so that there's a single "form library".

Reply:
Thank you, this is done.